src/components/ReactTags.js

```diff
@@ -387,7 +387,7 @@ class ReactTags extends Component {
     return tags.map((tag, index) => {
       return (
         <Tag
-          key={`${tag.id}-${index}`}
+          key={tag.key || tag.id}
```

Comment:
can you reverse this, `tag.id || tag.key` since id is getting used to check if the tags are unique

Reply:
Wouldn't it be better to use `key` since that's how react usually determines uniqueness in general?
Especially if we make it an optional property of each `tag` item